script/check-simulation-build-autotools

```diff
@@ -95,6 +95,7 @@ build_all_features()
 
     local options_1_3=(
         "-DOPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE=1"
+        "-DOPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE=1"
```

Comment:
Yes, we need `OPENTHREAD_CONFIG_NAT64_MANAGER `

`-DOPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE=1` should be placed in `options` since NAT64 is a 1.3 feature while border router is not an 1.3 only feature